server/cmd/api/api/playwright.go

```diff
@@ -0,0 +1,98 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os/exec"
+	"time"
+
+	"github.com/onkernel/kernel-images/server/lib/logger"
+)
+
+type ExecutePlaywrightRequest struct {
+	Code       string `json:"code"`
+	TimeoutSec *int   `json:"timeout_sec,omitempty"`
+}
+
+type ExecutePlaywrightResult struct {
+	Success bool        `json:"success"`
+	Result  interface{} `json:"result,omitempty"`
+	Error   string      `json:"error,omitempty"`
+	Stdout  string      `json:"stdout,omitempty"`
+	Stderr  string      `json:"stderr,omitempty"`
+}
+
+func (s *Service) ExecutePlaywrightCode(w http.ResponseWriter, r *http.Request) {
```

Comment:
@tembo after running `make oapi-generate`, there should be a new method for this endpoint on strictserverinterface in oapi.go, of which Service is an implementor of. use that